exec/java-exec/src/main/java/org/apache/drill/exec/util/record/RecordBatchStats.java

```diff
@@ -271,6 +271,19 @@ public static String printAllocatorStats(BufferAllocator allocator) {
     return msg.toString();
   }
 
+  /**
+   * Prints the configured batch size
+   *
+   * @param batchStatsContext batch stats context object
+   * @param batchSize contains the configured batch size
+   */
+  public static void printConfiguredBatchSize(RecordBatchStatsContext batchStatsContext,
+    int batchSize) {
+
```

Comment:
_logRecordBatchStats(...)_ does nothing if _batchStatsContext.isEnableBatchSzLogging()_ is false. The overhead of creating _message_ and the call overhead of _logRecordBatchStats(...)_ is wasted if that is the case. Please add a check and avoid the overhead.